src/main/java/mate/academy/spring/validation/FieldsValueMatch.java

```diff
@@ -0,0 +1,29 @@
+package mate.academy.spring.validation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import javax.validation.Constraint;
+import javax.validation.Payload;
+
+@Constraint(validatedBy = FieldsValueMatchValidator.class)
+@Target({ElementType.TYPE})
+@Retention(RetentionPolicy.RUNTIME)
+public @interface FieldsValueMatch {

```

Comment:
It’s an interesting solution but now you use this validator only once. It's can be simplified if you write validator only for the password (similar to Email Validator).

Reply:
Maybe I'll need this annotation later for another fields, not only passwords, so let it be)